torchci/lib/bot/index.ts

```diff
@@ -13,7 +13,7 @@ export default function bot(app: Probot) {
   autoCcBot(app);
   autoLabelBot(app);
   verifyDisableTestIssueBot(app);
-  ciflowBot(app);
+  // ciflowBot(app);
```

Comment:
why not just remove?

Reply:
just to make it easier to revert in case we need to